.packit.yaml

```diff
@@ -103,3 +103,26 @@ jobs:
       - fedora-latest # branched version, rawhide updates are created automatically
       - fedora-stable
       - epel-9
+
+  # vm image build
+  - job: copr_build
+    trigger: pull_request
+    targets:
+      - fedora-36-x86_64
+      - fedora-37-x86_64
+    project: vm-image-ogr-demo
+
+  - job: vm_image_build
+    trigger: pull_request
+    copr_chroot: fedora-36-x86_64
+    owner: packit
+    project: vm-image-ogr-demo
+    image_customizations:
+      packages: [python3-ogr]
```

Comment:
Do you actually need this, if the image is generated from the Copr build?

Reply:
Do you mean specify the package? I think yes but @TomasTomecek knows this better than me.

Reply:
Yup, the package. Agreed with you :D Though I'd expect this to be automatically installed because of the Copr build

Reply:
Yes we need this right now.

I guess we could default to all binary RPMs produced by the latest build though we don't have a mechanism right now for that. Hence it's explicit like this.